TRANSLATION_NOTES.asc

```diff
@@ -1,61 +1,54 @@
-== Hinweise zur deutschen Übersetzung
+= Hinweise zur deutschen Übersetzung
 
-<<<<<<< HEAD
-In diesem Dokument werden alle Punkte rund um die deutsche Übersetzung gesammelt. In diesem Dokument möchten wir festlegen, ob der Leser mit Du oder Sie angesprochen wird, ob der Erzähler die 1. Person Singular oder die 1. Person Plural verwendet, oder wie technische Fachbegriffe übersetzt werden sollen.
-=======
-After forking this repository to translate the work, this file is where the notes for coordinating the translation work would go.
-Things like standardizing on words and expressions so that the work is consistent or notes on how the contributing process is to be handled.
->>>>>>> 811dffebd4ff1348819f8acc70f954bd4ad65057
+In diesem Dokument werden alle Informationen rund um die deutsche Übersetzung gesammelt. Hiermit möchten wir beispielsweise festlegen, ob der Leser mit "Du" oder "Sie" angesprochen wird, ob der Erzähler die 1. Person Singular oder die 1. Person Plural verwendet oder wie technische Fachbegriffe übersetzt werden sollen.
```

Comment:
```suggestion
In diesem Dokument werden alle Informationen rund um die deutsche Übersetzung gesammelt. Hiermit möchten wir beispielsweise festlegen, ob der Leser mit „Du“ oder „Sie“ angesprochen wird, ob der Erzähler die 1. Person Singular oder die 1. Person Plural verwendet oder wie technische Fachbegriffe übersetzt werden sollen.
```

I opt for typographical quotes – would like to get feedback on this. It’s also worth adding a note about our decision to the relevant section on the book’s style guide for German.